legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/services/QueryMetricsHandler.java

```diff
@@ -69,4 +72,22 @@ public List<VersionQueryMetric> getSummaryByProjectVersion()
         });
         return all.values().stream().sorted(MOST_RECENTLY_QUERIED).collect(Collectors.toList());
     }
+
+    public void consolidateMetrics()
+    {
+        List<VersionQueryMetric> versionQueryMetric = getSummaryByProjectVersion();
```

Comment:
i know this is not part of the review but the summary function can be written better and easier to read using streams and a max operator..